Testscripts/Windows/PERF-SCHEDULER.ps1

```diff
@@ -0,0 +1,208 @@
+            $currentStatus = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password `
+                -command "tail -2 ${testName}ConsoleLogs.txt | head -1"
+            Write-LogInfo "Current Test Status : $currentStatus"
+            Wait-Time -seconds 20
+        }
+        $finalStatus = Run-LinuxCmd -ip $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password `
+            -command "cat ./state.txt"
+        Copy-RemoteFiles -downloadFrom $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password `
+            -download -downloadTo $LogDir -files "${testName}ConsoleLogs.txt"
+        Copy-RemoteFiles -downloadFrom $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password `
+            -download -downloadTo $LogDir -files "hackbench.*.log"
+        Copy-RemoteFiles -downloadFrom $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password `
+            -download -downloadTo $LogDir -files "schbench.*.log"
+        Copy-RemoteFiles -downloadFrom $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password `
+            -download -downloadTo $LogDir -files "results*.log, results*.csv"
+        Copy-RemoteFiles -downloadFrom $AllVMData.PublicIP -port $AllVMData.SSHPort -username $user -password $password `
+            -download -downloadTo $LogDir -files "VM_properties.csv"
+
+        ##################################################################################################################
+        #region Parse log to obtain the test results
+        ##################################################################################################################
+        $uploadResults = $true
+        $hackbench_results = Get-Content -Path "$LogDir\results_hackbench.log"
+        $schbench_results = Get-Content -Path "$LogDir\results_schbench.log"
+
+        $connResult = "======================================="
+        $metadata = "Hackbench test results"
+        $currentTestResult.TestSummary += New-ResultSummary -testResult $connResult -metaData $metadata -checkValues "PASS,FAIL,ABORTED" -testName $currentTestData.testName
+        foreach ($line in $hackbench_results) {
+            if ($line -imatch "TestMode") {
+                continue;
+            }
+            try {
+                $datasize_bytes = ($line.Trim() -Replace " +"," ").Split(" ")[1]
+                $hackbenchType = ($line.Trim() -Replace " +"," ").Split(" ")[5]
+                $latency_sec = ($line.Trim() -Replace " +"," ").Split(" ")[6]
+                $connResult = "DataSize_bytes=$datasize_bytes Latency_sec=$latency_sec"
+                $metadata = "HackbenchType=$hackbenchType"
+                $currentTestResult.TestSummary += New-ResultSummary -testResult $connResult -metaData $metadata -checkValues "PASS,FAIL,ABORTED" -testName $currentTestData.testName
```

Comment:
All those Summary text will be save into telemetry database, but if you need further reference those data for some report, suggest leave the last number value for Latency_sec with this format:   xxxxxx,yyy,zzz : 4.551
xxxxxx,yyy,zzz is metadata,  4.551 is the value  in  New-ResultSummary()

Reply:
If xxxxxx,yyy,zzz is metadata, 4.551 is the value in New-ResultSummary(), the output format is xxxxxx,yyy,zzz =4.551. I think this format is good.